tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -187,6 +187,8 @@ spec:
         - "appstudio=$(results.TEST_OUTPUT.path)"
         - "--output"
         - "json=$(params.HOMEDIR)/report-json.json"
+        - "2>"
+        - "debug.log"
```

Comment:
We can't use shell redirects here, these are arguments passed to `ec` directly, not a shell invocation...